matchminer/custom.py

```diff
@@ -1423,12 +1424,13 @@ def run_ctims_matchengine_job(trial_internal_ids):
             db_name='matchminer',
             ignore_run_log=True,
             ignore_report_date=True,
-            protocol_nos=trial_internal_ids
+            protocol_nos=trial_internal_ids,
+            failed_protocol_nos={}
     ) as me:
         me.get_matches_for_all_trials()
-        me.update_all_matches()
+        if (len(me.failed_protocol_nos.keys()) > 0):
+            return me.failed_protocol_nos
```

Comment:
what happens to the partial success? It won't update_all_matches() for partial success ones

Reply:
good catch.